server/bundler/babel/babel-lodash-es/test/index.js

```diff
@@ -3,13 +3,15 @@
 /**
  * External dependencies
  */
-const babel = require( 'babel-core' );
+const babel = require( '@babel/core' );
 
 describe( 'babel-lodash-es', () => {
 	function transform( code ) {
-		return babel.transform( code, {
-			plugins: [ require( '..' ) ],
-		} ).code;
+		return babel
+			.transform( code, {
+				plugins: [ require( '..' ) ],
+			} )
+			.code.replace( /\"/g, "'" );
```

Comment:
this was strange and I'd like to investigate more. For some reason babel was using double quotes instead of single quotes which causes funny escaping  + breaks the test